x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/dimension_container.scss

```diff
@@ -6,11 +6,15 @@
   @include euiFlyout;
   // But with custom positioning to keep it within the sidebar contents
   position: absolute;
-  right: 0;
   left: 0;
-  top: 0;
-  bottom: 0;
   animation: euiFlyout $euiAnimSpeedNormal $euiAnimSlightResistance;
+  @include euiBreakpoint('l', 'xl') {
+    top: 0 !important;
+    height: 100% !important;
+  }
+  @include euiBreakpoint('xs', 's', 'm') {
+    @include euiFlyout;
+  }
```

Comment:
I ran into a similar issue here with the flyout being part of the right side container, but we should really disable scrolling while the flyout is open. I think this is something the EUI components can do for us?

<img width="378" alt="Screen Shot 2021-04-22 at 4 36 32 PM" src="https://user-images.githubusercontent.com/666475/115781934-0aaeb180-a389-11eb-8ad0-9ae09f0281c5.png">